crates/mod-host-assets/Cargo.toml

```diff
@@ -13,6 +13,7 @@ from-singleton = { version = "2", features = ["regex-unicode"] }
 undname = "2.1"
 pelite = "0.10"
 regex = "1"
+normpath = "1.3.0"
```

Comment:
Mind adding this to workspace dependencies and updating me3-cli's Cargo.toml, too? We also use it there.